src/System.Net.Http.Formatting.NetStandard/System.Net.Http.Formatting.NetStandard.csproj

```diff
@@ -59,6 +59,9 @@
     <Compile Include="..\System.Net.Http.Formatting\Formatting\FormUrlEncodedJson.cs">
       <Link>Formatting\FormUrlEncodedJson.cs</Link>
     </Compile>
+    <Compile Include="..\System.Net.Http.Formatting\Formatting\FormUrlEncodedMediaTypeFormatter.cs">
+      <Link>Formatting\FormUrlEncodedMediaTypeFormatter.cs</Link>
+    </Compile>
```

Comment:
@YakhontovYaroslav please edit the System.Net.Http.Formatting.NetCore.csproj file and add the same link. Then, confirm everything builds from the command line. (I suspect problems building in VS 2017 will go away as I complete work on the infrastructure.)

Reply:
Add similar links for `FormUrlEncodedMediaTypeFormatterTests` in the test projects. That may require building `DerivedFormUrlEncodedMediaTypeFormatter` too.